woocommerce/woocommerce-ios
# Select order when opening via deep link

Part of #11674 


## Description

- continuation of https://github.com/woocommerce/woocommerce-ios/pull/11718
- add logic to select the order cell in list of orders

## Testing instructions

- same instructions as in https://github.com/woocommerce/woocommerce-ios/pull/11718, just check if the correct order cell is selected in split view

## Screenshots


As you can see before the opened order was not selected in the orders list:

| Before      | After |
| ----------- | ----------- |
| ![Simulator Screen Recording - iPad Pro (11-inch) (4th generation) - 2024-01-18 at 16 01 33](https://github.com/woocommerce/woocommerce-ios/assets/6242034/3e496bf9-8f40-467f-8579-0ce37b8d7c89)      | ![Simulator Screen Recording - iPad Pro (11-inch) (4th generation) - 2024-01-18 at 15 58 56](https://github.com/woocommerce/woocommerce-ios/assets/6242034/43c54aaa-4982-41b4-a050-57f070c2157d)       |




---
